### PR TITLE
Remove Pending Loans [PAR-76]

### PR DESCRIPTION
**Pull request by vastImmortalSuns**
While this is mainly a backend PR, I included everyone because it also showcases what data should be sent to the backend for updating/inserting a new loan.

Lemme know if you have any questions!

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -424,37 +424,41 @@ var loans = (function () {
...
         var insertQuery =
-          "INSERT INTO loans book_id, requester_contact, owner_contact, status) " +
-          "VALUES (?,?,?,?)";
-        var insertParams = [
-          loan.book_id,
-          loan.requester_contact,
-          " ",
-          "pending",
-        ];
+          "INSERT INTO loans (book_id, requester_id, owner_contact, status) " +
```
Comment (lemniscate8) [inline db.js]: Is `requester_contact` still a required field in the database?

Comment (Hashiphoto) [reply db.js]: `owner_contact` and `requester_contact` are still NOT NULL fields, so I would think that you would have to insert a value for both of them?

File: db.js [hunk 2]
```diff
@@ -424,37 +424,41 @@ var loans = (function () {
...
-      var inserts = [
-        loan.book_id,
-        loan.requester_contact,
-        " ",
-        loan.status,
-        loan.id,
-      ];
-      updateQuery = mysql.format(updateQuery, inserts);
+    updateLoanById: async function (loan) {
+      if (loan.status === "accepted") {
```
Comment (Hashiphoto) [inline db.js:442]: With this restriction in place, how would you ever change a loan to "loaned" or "late"?

Comment (vastImmortalSuns) [reply db.js:442]: I was thinking of that yesterday. My plan was to merge this PR as it has its own distinct card attached to it, and then buff out the rest of the loans routes with the appropriate logic for the other tasks required of them. There's another card in Jira called "fix loans for real," so I inferred that those other tasks fell within the realm of that card.

Comment (Hashiphoto) [reply db.js:442]: Okay, sounds good. I'm most interested in getting the loans route fixed asap, and if you have a plan to make that happen, I'm all for it.

Comment (vastImmortalSuns) [reply db.js:442]: Cool, I'll go ahead and merge

**Pull request merged**